topologies/hipster-shop.json

```diff
@@ -15,7 +15,7 @@
             "downstreamCalls" : { "productcatalogservice" : "/GetProducts", "recommendationservice" : "/GetRecommendations", "adservice" : "/AdRequest" },
             "maxLatencyMillis": 200,
             "tagSets": [
-              { "weight": 1, "tags": { "starter" : "charmander"}},
+              { "weight": 1, "tags": { "starter" : "charmander"}, "tagGenerators":  [{"name": "3000-val-tag", "numTags":  50, "numVals":  3000}]},
```

Comment:
one more feature request since we are here - maybe even allow a knob for average tag-value length to simulate extremely long tag values (e.g. 500 bytes) - when this is specified probably the value could be a true junk

Reply:
Feel free to do it in a separate change or say out of scope

Reply:
Done.